README.md

```diff
@@ -1,16 +1,248 @@
-# Title
+# Methods in Ruby
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Describe routines and procedures for our code using methods
+- Define a method
+- Recognize how to call methods
 
 ## Introduction
 
-## SWBAT 1
+A method in Ruby is a _block_ of code that returns a value. With methods, we can
+organize our code into chunks of reusable functionality that can be easily used
+by other areas of their program.
 
-## SWBAT 2
+Ruby methods are very similar to functions in any other programming language.
+Ruby methods are used to bundle one or more repeatable statements into a single
+unit.
+
+In this lesson we'll introduce methods, distinguish them from data types, and
+cover how to create and execute them in your Ruby program.
+
+## Describe Routines and Procedures for Our Code Using Methods
+
+Methods define a new thing that your program can do. Variables in Ruby store
+data; methods store a new routine or behavior your program can use. Variables
+are like "nouns", a thing, and methods are like "verbs", an action.
+
+For example, imagine needing to say "Hello World!" five times. It could look
+like this:
```

Comment:
I like this example, but maybe go even lower fi to start, just write out `puts "Hello World!"` five times.